addons/mission/functions/fnc_createEquipment.sqf

```diff
@@ -0,0 +1,55 @@
+#include "..\script_component.hpp"
+/*
+ *	Author: D.Anderson
+ *	Creates Group in Roster Subject
+ *	
+ *	Arguments:
+ *	0: The Unit <OBJECT>
+ *	
+ *	Return Value:
+ *	String
+ *	
+ *	Example:
+ *	[player] call dAn_fnc_createGroup
+ *	
+ * 	Public: No
+ */
+
+params[["_unit", ACE_player], '_details'];
+
+private _grpArr = []; 
+
+private _grpStr = [
+	'<font face="RobotoCondensedBold" size="18">',
+	groupId group _unit,
+	'</font>',
+	'<br/>'
+] joinString "";
+
+_grpArr pushBack _grpStr;
+
+if (isPlayer _unit) then {
+	private _unitIdentity = [_unit] call FUNC(getIdentity);
+	private _weaponsText = [_unit] call FUNC(getWeapons);
+	private _equipmentText = [_unit, _details] call FUNC(getEquipment);
+
+	private _grpText = [
+		'<img title="', _unitIdentity select 0, '" src="', _unitIdentity select 1, '" height="16"/>',
```

Comment:
_unitIdentity select x could be replaced if we add
_unitIdentity params ["_firstParam", "_secondParam"];
ofc all of them should have easy to understand name (cosmetic change for easier understanding whats added in there)